# Exercise: exceptions


Take the earlier exercise where we used Test::Exception and see how to rewrite it using
Test::Fatal and/or Test::Trap.


